packages-content-model/roosterjs-content-model-core/lib/corePlugin/SelectionPlugin.ts

```diff
@@ -221,6 +223,18 @@ class SelectionPlugin implements PluginWithState<SelectionPluginState> {
             if (newSelection?.type == 'range') {
                 this.state.selection = newSelection;
             }
+
```

Comment:
actually, I mean for line 223-225, it should only be run for Safari. Then for 227-235, it should be run for all

Reply:
Please refer to comment at line 219

Reply:
And this part should also not run when suspendSelectionChangeEvent.

Better add this check at beginning of this function